CHANGELOG.develop

```diff
@@ -423,6 +423,17 @@ Benner and Paweł Siudak):
 - Renamed =spacemacs/mplist-get= function to =spacemacs/mplist-get-values=,
   renamed =spacemacs/plist-get= function to =spacemacs/mplist-get-value=, and
   refactored the latter function (thanks to Benjamin Reynolds)
+- Allow spaces and empty values in environment variable values (thanks to
+  Sylvain Benner)
+- Add support for fetching of environment variables on Microsoft Windows
+  (only tested on Windows 10) (thanks to Sylvain Benner)
```

Comment:
Caching environment variables is a new feature since the last release, so I don't think these entries are needed.